jbpm-flow-builder/src/test/java/org/jbpm/process/workitem/WorkDefinitionImplTest.java

```diff
@@ -122,7 +122,7 @@ public void testParameterValuesWithEnumsOnly() throws Exception {
             if (entry.getKey().equals("param1")) {
                 EnumDataType paramEnum = (EnumDataType) entry.getValue();
                 assertEquals("org.jbpm.process.workitem.enums.AnimalsEnum", paramEnum.getClassName());
-                Map<String, Object> paramValuesMap = paramEnum.getValueMap();
+                Map<String, Object> paramValuesMap = paramEnum.getValueMap(null);
```

Comment:
I understand this is just quick fix to make the build green again, but passing `null` as an argument into into the method seems weird.